setup.py

```diff
@@ -104,5 +104,5 @@
     tests_require=TESTS_REQUIRE,
     url=URL,
     packages=find_packages(),
-    package_data={"xarray": ["py.typed", "tests/data/*"]},
+    package_data={"xarray": ["py.typed", "tests/data/*", "static/*"]},
```

Comment:
This might need `static/**` to pick up subdirectories. When I tried installing with pip it didn't work otherwise.